__tests__/app.test.js

```diff
@@ -22,8 +22,27 @@ describe("GET api/topics", () => {
       .expect(404)
       .end((err, res) => {
         if (err) return done(err);
-        console.log(err);
         done();
       });
   });
+  test("Expect length of topics array to be greater than one", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBeGreaterThan(1);
```

Comment:
I would personally include this at the same point you're iterating over the array at the next test (ie. both expect statements one after the other) since you're making two different requests to the api, you would be making sure that the result of that specific api call isn't an empty array